ocs_ci/ocs/cluster.py

```diff
@@ -644,6 +644,47 @@ def get_rebalance_status(self):
                 and states['count'] == total_pg_count
             )
 
+    def wait_for_rebalance(self, timeout=None):
+        """
+        Wait for re-balance to complete
+
+        Args:
+            timeout (int): Time to wait for the completion of re-balance.
+                In case IO is running in the test background (according to --io-in-bg
+                run-ci param, the timeout will be determined by the amount of load
+                (io_load run-ci param)
+
+        Returns:
+            bool: True if rebalance completed, False otherwise
+
+        """
+        if config.RUN.get('io_in_bg'):
+            io_load = config.RUN.get('io_load')
```

Comment:
is this value by default 0? If not please use default value here in get to have 0